h5glance/utils.py

```diff
@@ -0,0 +1,54 @@
+"""Helper functions
+"""
+import h5py
+
+_mapping = {}
+_mapping[h5py.File] = "file"
+_mapping[h5py.Group] = "group"
+_mapping[h5py.Dataset] = "dataset"
```

Comment:
Why not write the initial mapping with these in?

```python
_mapping = {
    h5py.File: 'file',
...
```

Reply:
It's up to you. I updated as you requested.